etc/helm/examples/enterprise-dev.yaml

```diff
@@ -9,4 +9,17 @@ enterpriseServer:
     tag: local
 pachd:
   enabled: false
-
+  activateEnterprise: false
+  enterpriseLicenseKey: ""
```

Comment:
Won't this fail without a license key?

Reply:
as noted in another PR comment, `activateEnterprise == true && enterpriseLicenseKey == "" is implying registering with another cluster here`
https://github.com/pachyderm/pachyderm/pull/6838/files#r711054238